auth0_flutter/ios/Classes/WebAuth/WebAuthModels.swift

```diff
@@ -0,0 +1,15 @@
+enum SafariViewControllerProperty: String, CaseIterable {
+    case presentationStyle
+}
+
+struct SafariViewController {
+    var presentationStyle: UIModalPresentationStyle = UIModalPresentationStyle.fullScreen
```

Comment:
The default value here is not necessary because it's being set in the initializer.